pkgs/development/libraries/arcan/default.nix

```diff
@@ -0,0 +1,185 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, xz
+, libuvc
+, libffi
+, libX11
+, libxcb
+, libXcomposite
+, libXfixes
+, xcbutil
+, xcbutilwm
+, libXau
+, libXdmcp
+, openal
+, freetype
+, sqlite
+, libGL
+, libdrm
+, libxkbcommon
+, mesa
+, valgrind
+, luajit
+, libusb1
+, harfbuzzFull
+, SDL2
+
+# For generating API documentation
+, ruby
+
+#
+, withVLC ? true
+, libvlc
+
+#
+, withFFmpeg ? true
+, ffmpeg-full
+
+#
+, withWayland ? true
+, wayland
+, wayland-protocols
+
+#
+, withEspeak ? true
+, espeak-classic
+
+#
+, withLibmagic ? true
+, file
+
+#
+, withTesseractAndLeptonica ? true
```

Comment:
```suggestion
, SDL2
, ruby
, withVLC ? true
, libvlc
, withFFmpeg ? true
, ffmpeg-full
, withWayland ? true
, wayland
, wayland-protocols
, withEspeak ? true
, espeak-classic
, withLibmagic ? true
, file
, withTesseractAndLeptonica ? true
```

Reply:
Why don't you want documentation?

Reply:
The names are self describing and we normally don't document things that are clear. Also this stretches the file quite a bit.

Reply:
What is clear what the different dependencies enables in Arcan? Or shouldn't Nixpkgs contain any documentation about the different packages? If yes, why? Documentation is good.